run_tests.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
```

Comment:
This file isn't necessary for the unit tests as well, but good to have for the integration tests.